index.html

```diff
@@ -53,6 +54,241 @@ <h1>I'm Redwan.<br/>
+                          <div class="client-parent">
+                            <ul>
+                                <li>CANOPY</li>
+                                <li>Back End Dev </li>
+                                <li>2015</li>
+                            </ul>
+                          </div>
+                          <p>
+                            A daily selection of privately <br/>
+                            personalized reads; no accounts or <br/>
+                            sign-ups required.
+                          </p>     
+                        </div>
+                        <div class="tags">
+                            <ul>
+                                <li>
+                                    <a class="btn" href="">HTML</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">CSS</a>
+                                </li>
+                                <li>
+                                    <a class="btn" href="">JavaScript</a>
+                                </li>
+                            </ul>
+                          </div>
+                          <div class="action">
+                            <button class="submit">See project</button>
+                        </div>
+                    </div>
+                </div>
+               </div>
+        </section>
+
+
+        <section class="section-bottom"> 
+            <div class="headerTop">
+                <h1>About Myself </h1>
+                <p >
+                    Hello I’m a software developer! I can 
+                    help you build a product , feature or 
+                    website. Look through some of my work 
+                    and experience! If you like what you see 
+                    and have a project you need coded, 
+                    don’t hesitate to contact me.
+                </p>
+            </div>
+            <!-- End of header top -->
+            <div class="headerBottom">
+                <a href="#">LET’S CONNECT</a>
+                <div class="link-btn">
+                    <ul>
+                        <li>
+                            <a href="#"><img src="images/link-Vector4.svg" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Instagram2.png" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/link-Vector5.svg" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Twitter.svg" alt="Twitter"/></a>
+                        </li>
+                        
+                        <li>
+                            <a href="#"><img src="images/link-Vector3.svg" alt=""/></a>
+                        </li>
+                        
+                       
```

Comment:
- [ ] Kindly reduce the horizontal spacing in your design so it would match the design from Figma 🙏🏽 Note that it is essential to have keen eyes on details. This is an opportunity to practice that skill ❤️ 

Your design 👇🏽 

![Screenshot from 2023-06-06 01-22-26](https://github.com/ethred/Portfolio/assets/10439283/a3842717-19e2-4c36-a5d1-b13f251c7d0e)

Design from Figma 👇🏽 

![Screenshot from 2023-06-06 01-22-42](https://github.com/ethred/Portfolio/assets/10439283/2f2f82e7-d942-4672-892e-44b3600fca26)

Reply:
thanks a-lot for all your comments and update the file based on the comments